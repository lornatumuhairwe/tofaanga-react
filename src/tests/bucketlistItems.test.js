import React from 'react';
import renderer from 'react-test-renderer';
import BucketlistItems from "../components/Bucketlist/bucketlistItems";

describe('bucketlist items snapshot', () => {
    it('matches snapshot', () => {
            const blItems = renderer.create(<BucketlistItems/>);
            expect(blItems.toJSON()).toMatchSnapshot();
        },

    );
});
