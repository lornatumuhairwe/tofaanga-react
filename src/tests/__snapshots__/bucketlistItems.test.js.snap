// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bucketlist items snapshot matches snapshot 1`] = `
<div
  className="panel"
>
  <div
    className="panel panel-info"
    id={null}
  >
    <div
      aria-expanded={null}
      aria-hidden={true}
      className="panel-collapse collapse"
      id={undefined}
      role={undefined}
    >
      <div
        className="panel-body"
      >
        <table
          className="table"
        >
          <thead>
            <tr>
              <th>
                Item
              </th>
              <th>
                Deadline
              </th>
              <th>
                Status
              </th>
            </tr>
          </thead>
          <tbody />
        </table>
      </div>
    </div>
  </div>
</div>
`;
